websites-css/master.css

```diff
@@ -10,8 +10,8 @@
   --color-content-section: #3a3a4d;
   --font-size-header: 1.2rem;
   --font-size-h2: 2.5rem;
-  --font-size-h3: 1.8rem;
-  --font-size-p: 1.2rem;
+  --font-size-h3: 1.5rem;
+  --font-size-p: 1rem;
```

Comment:
Putting 1rem is not necessary unless we specify another font-size in the root.

example: html {
font-size: 20px
}
This means that writing 1rem only changes font size if you actually changed it in its root, and in this case 1rem would be 20px, 2rem would be 40px etc.